src/kvirc/ui/KviMenuBar.cpp

```diff
@@ -147,21 +147,6 @@ void KviMenuBar::setupHelpPopup(QMenu *pop)
 	help->addSeparator();
 	pAction = help->addAction(*(g_pIconManager->getSmallIcon(KviIconManager::Channel)),__tr2qs("Join KVIrc International Channel on freenode"));
```

Comment:
I guess we can remove the `International` then, might replace it with `Support`? and also drop `on Freenode` since there is only one now?